website/docs/components/app-header/partials/code/how-to-use.md

```diff
@@ -0,0 +1,167 @@
+
+
+```handlebars
+<Hds::AppHeader>
+  <:globalItemsBefore>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HomeLink" @background="#e4e4e4" />
+  </:globalItemsBefore>
+
+  <:globalItemsAfter>
+    <Doc::Placeholder @height="2em" @width="auto" @text="OrgSwitcher" @background="#e4e4e4" />
+  </:globalItemsAfter>
+
+  <:utilityItems>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HelpMenu" @background="#e4e4e4" />
+    <Doc::Placeholder @height="2em" @width="auto" @text="UserMenu" @background="#e4e4e4" />
+  </:utilityItems>
+</Hds::AppHeader>
+```
+
+### Content
+
+#### HomeLink
+
+The `Hds::AppHeader::HomeLink` child component should be yielded within the `<:globalItemsBefore>` block. It provides consistent branding and navigates the user to the “home“ or main dashboard page.
+
+It requires a value for the `@icon` and `@ariaLabel` arguments.
+
+It is built on top of the [`Hds::Interactive` component](/utilities/interactive), so it accepts all its routing arguments (eg. `@href`, `@route`, `@query`, `@model(s)`, etc.).
+
+Refer to the [Component API section](/components/app-header?tab=code#appheaderhomelink) for details.
+
+```handlebars
+<Hds::AppHeader>
+  <:globalItemsBefore>
+    <Hds::AppHeader::HomeLink 
+      @icon="hashicorp" 
+      @ariaLabel="HashiCorp home menu"
+      @href="/"
+    />
+  </:globalItemsBefore>
+  
+  <:globalItemsAfter>
+    <Doc::Placeholder @height="2em" @width="auto" @text="OrgSwitcher" @background="#e4e4e4" />
+  </:globalItemsAfter>
+  <:utilityItems>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HelpMenu" @background="#e4e4e4" />
+    <Doc::Placeholder @height="2em" @width="auto" @text="UserMenu" @background="#e4e4e4" />
+  </:utilityItems>
+</Hds::AppHeader>
+```
+
+It also accepts optional arguments; for example, it’s possible to provide a custom color for the icon if needed:
+
+```handlebars
+<Hds::AppHeader>
+  <:globalItemsBefore>
+    <Hds::AppHeader::HomeLink 
+      @icon="terraform" 
+      @ariaLabel="Terraform home menu"
+      @color="var(--token-color-terraform-brand)"
+      @href="/"
+    />
+  </:globalItemsBefore>
+  
+  <:globalItemsAfter>
+    <Doc::Placeholder @height="2em" @width="auto" @text="OrgSwitcher" @background="#e4e4e4" />
+  </:globalItemsAfter>
+
+  <:utilityItems>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HelpMenu" @background="#e4e4e4" />
+    <Doc::Placeholder @height="2em" @width="auto" @text="UserMenu" @background="#e4e4e4" />
+  </:utilityItems>
+</Hds::AppHeader>
+```
+
+#### “Context switcher” (eg. organization or project selector)
+
+Consumers should provide their own “context switcher” (eg. organization or project selector) control yielded within the `<:globalItemsAfter>` block. HDS does not currently provide this component.
```

Comment:
In the guidelines I included a note about this; "For implementation in code, use the HDS Dropdown or, if necessary, create your own custom component styled after the Dropdown".

Should we align a bit more on the language we use here?

Reply:
I've been going through and updating to use this language:

“context switcher” (e.g., “org switcher” or “project switcher”) control

Reply:
Just pushed up new edits